docs/docs/api/classes/Bluehawk.md

```diff
@@ -0,0 +1,198 @@
+---
+id: "Bluehawk"
```

Comment:
Do these files actually need to be committed or can they be added to gitignore and just generated on the fly (during github action)?

Reply:
i think .gitignore-ing them would make sense. will make that change.

Reply:
done